exchangeAPI/src/index.js

```diff
@@ -0,0 +1,65 @@
+document.querySelector("#send").onclick = function (e) {
+  e.preventDefault();
+  const symbol = document.querySelector("#symbol").value;
+  const date = document.querySelector("#date").value;
+  const symbolErrors = validateSymbol(symbol);
+  const dateErrors = validateDate(date);
+  handleErrors(symbolErrors, dateErrors);
+  if (!symbolErrors && !dateErrors) {
+    removeLoading();
+    removeOldSymbols();
+    handlerDataSymbols(symbol, date);
+  }
+};
+const removeLoading = () => {
+  const loading = document.querySelector(".loading");
+  loading.classList.remove("hidden");
+};
+
+const removeOldSymbols = () => {
+  const $list = document.querySelectorAll(".item");
+  if ($list)
+    $list.forEach(function ($item) {
+      $item.remove();
+    });
+};
+function handlerDataSymbols(symbol, date) {
```

Comment:
Avoid using the word data and I think there's a typo instead of `handler` it should be `handleSymbols`.